### ISSUE TITLE
[Bug]: Negative counter for manually scheduled reindexes

### ISSUE THREAD

**Issue by p0358** [tubearchivist/tubearchivist#478]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

TrueNAS Scale

## Describe the bug
![obraz](https://github.com/tubearchivist/tubearchivist/assets/5182588/046c645a-96d7-42b5-a7fa-4ea643f46885)
The negative counter goes down to 0.

## Steps To Reproduce
1. Open a few videos in a few tabs.
2. Click "Reindex" button on each.
3. Go to Settings tab where you can see the progress.

## Expected behavior
It should probably be showing the videos on the format of like `{current_video}/{all_queued_videos}` rather than `-{current_video}/1`.

This is honestly a very minor aesthetic issue, but I figured it might still be worth to report it...

### Relevant log output

```shell
[I removed video IDs manually before pasting]

2023-05-21 18:54:32.166157+00:00[2023-05-21 20:54:32,165: INFO/MainProcess] Task check_reindex[4b0570d4-bb99-40e5-ac24-523aff6e24c8] received
2023-05-21 18:54:32.167562+00:00[2023-05-21 20:54:32,166: WARNING/ForkPoolWorker-4] check_reindex create callback
2023-05-21 18:54:32.170095+00:00[2023-05-21 20:54:32,169: WARNING/ForkPoolWorker-4] [task][check_reindex] reindex {'video': ['AAAAAAAAAA']}
2023-05-21 18:54:32.181669+00:00[2023-05-21 20:54:32,181: WARNING/ForkPoolWorker-4] validating cookie
2023-05-21 18:54:32.372412+00:00[2023-05-21 20:54:32,372: WARNING/ForkPoolWorker-4] AAAAAAAAAA: get metadata from es
2023-05-21 18:54:32.377180+00:00[2023-05-21 20:54:32,376: WARNING/ForkPoolWorker-4] AAAAAAAAAA: get metadata from youtube
2023-05-21 18:54:33.787240+00:00[2023-05-21 20:54:33,786: WARNING/ForkPoolWorker-4] UCCCCCCCCCCCCCCCCCCCC: get metadata from es
2023-05-21 18:54:33.920614+00:00[2023-05-21 20:54:33,920: WARNING/ForkPoolWorker-4] AAAAAAAAAA: get sponsorblock timestamps
2023-05-21 18:54:34.108097+00:00[2023-05-21 20:54:34,107: WARNING/ForkPoolWorker-4] AAAAAAAAAA: sponsorblock failed: 404
2023-05-21 18:54:34.109759+00:00[2023-05-21 20:54:34,109: WARNING/ForkPoolWorker-4] AAAAAAAAAA: delete subtitles
2023-05-21 18:54:34.110017+00:00[2023-05-21 20:54:34,109: WARNING/ForkPoolWorker-4] AAAAAAAAAA-en: get user uploaded subtitles
2023-05-21 18:54:34.110120+00:00[2023-05-21 20:54:34,109: WARNING/ForkPoolWorker-4] AAAAAAAAAA-pl: get user uploaded subtitles
2023-05-21 18:54:34.110217+00:00[2023-05-21 20:54:34,109: WARNING/ForkPoolWorker-4] AAAAAAAAAA-kr: get user uploaded subtitles
2023-05-21 18:54:34.110252+00:00[2023-05-21 20:54:34,110: WARNING/ForkPoolWorker-4] AAAAAAAAAA-jp: get user uploaded subtitles
2023-05-21 18:54:34.110324+00:00[2023-05-21 20:54:34,110: WARNING/ForkPoolWorker-4] AAAAAAAAAA-ru: get user uploaded subtitles
2023-05-21 18:54:34.263102+00:00[2023-05-21 20:54:34,262: WARNING/ForkPoolWorker-4] AAAAAAAAAA: get comments
2023-05-21 18:54:34.802719+00:00[2023-05-21 20:54:34,802: INFO/MainProcess] Task check_reindex[f1adbb2c-b941-4b1f-a774-85451a182e3a] received
2023-05-21 18:54:34.804220+00:00[2023-05-21 20:54:34,803: WARNING/ForkPoolWorker-2] check_reindex create callback
2023-05-21 18:54:34.807139+00:00[2023-05-21 20:54:34,806: WARNING/ForkPoolWorker-2] [task][check_reindex] reindex {'video': ['BBBBBBBBBB']}
2023-05-21 18:54:34.813579+00:00[2023-05-21 20:54:34,813: WARNING/ForkPoolWorker-2] [task][check_reindex] reindex queue is already running
2023-05-21 18:54:34.817116+00:00[2023-05-21 20:54:34,816: WARNING/ForkPoolWorker-2] f1adbb2c-b941-4b1f-a774-85451a182e3a success callback
2023-05-21 18:54:34.819115+00:00[2023-05-21 20:54:34,818: INFO/ForkPoolWorker-2] Task check_reindex[f1adbb2c-b941-4b1f-a774-85451a182e3a] succeeded in 0.01343067898415029s: None
2023-05-21 18:54:37.060521+00:00[2023-05-21 20:54:37,060: WARNING/ForkPoolWorker-4] {"_index":"ta_comment","_id":"AAAAAAAAAA","found":false}
2023-05-21 18:54:37.060586+00:00[2023-05-21 20:54:37,060: WARNING/ForkPoolWorker-4] comments: not found AAAAAAAAAA
2023-05-21 18:54:37.060641+00:00[2023-05-21 20:54:37,060: WARNING/ForkPoolWorker-4] AAAAAAAAAA: delete comments
2023-05-21 18:54:37.084477+00:00[2023-05-21 20:54:37,084: INFO/MainProcess] Task check_reindex[0ecc6c41-e40b-4354-a30b-6cf493093001] received
2023-05-21 18:54:37.087066+00:00[2023-05-21 20:54:37,086: WARNING/ForkPoolWorker-2] check_reindex create callback
2023-05-21 18:54:37.088758+00:00[2023-05-21 20:54:37,088: WARNING/ForkPoolWorker-2] [task][check_reindex] reindex {'video': ['VpcF5s1m2uY']}
2023-05-21 18:54:37.096055+00:00[2023-05-21 20:54:37,095: WARNING/ForkPoolWorker-2] [task][check_reindex] reindex queue is already running
2023-05-21 18:54:37.098497+00:00[2023-05-21 20:54:37,098: WARNING/ForkPoolWorker-2] 0ecc6c41-e40b-4354-a30b-6cf493093001 success callback
2023-05-21 18:54:37.101319+00:00[2023-05-21 20:54:37,101: INFO/ForkPoolWorker-2] Task check_reindex[0ecc6c41-e40b-4354-a30b-6cf493093001] succeeded in 0.012459899997338653s: None
2023-05-21 18:54:37.105853+00:00[2023-05-21 20:54:37,105: WARNING/ForkPoolWorker-4] {"_index":"ta_comment","_id":"AAAAAAAAAA","_version":1,"result":"not_found","forced_refresh":true,"_shards":{"total":1,"successful":1,"failed":0},"_seq_no":2011,"_primary_term":13}
2023-05-21 18:54:37.106030+00:00[2023-05-21 20:54:37,105: WARNING/ForkPoolWorker-4] AAAAAAAAAA: upload comments
2023-05-21 18:54:40.138832+00:00[2023-05-21 20:54:40,138: WARNING/ForkPoolWorker-4] BBBBBBBBBB: get metadata from es
2023-05-21 18:54:40.144135+00:00[2023-05-21 20:54:40,143: WARNING/ForkPoolWorker-4] BBBBBBBBBB: get metadata from youtube
```

edit: another screenshot example:
![image](https://github.com/tubearchivist/tubearchivist/assets/5182588/00e512b4-824c-4d59-9bbf-80bc82c45d49)

**Comment by bbilly1** [comment]
yeah, can reproduce this too. Probably something silly. :-) Do you want to look into it?

**Comment by bbilly1** [comment]
should now be fixed in v0.4.0.
